django/contrib/admin/sites.py

```diff
@@ -63,6 +64,21 @@ def __init__(self, name='admin'):
         self.name = name
         self._actions = {'delete_selected': actions.delete_selected}
         self._global_actions = self._actions.copy()
+        all_sites.add(self)
+
+    def check(self):
+        """
+        Run the system checks of the underlying admin objects.
+        """
+        if not settings.DEBUG:
+            return []
+
+        errors = []
+        admin_objects_to_check = (o for o in self._registry.values() if type(o) is not ModelAdmin)
```

Comment:
I think that's a good name if the code were more complicated but `modeladmins` and `for modeladmin in modeladmins:` seems okay to me.